packages/es-components/src/components/patterns/datepicker/DatePicker.specs.js

```diff
@@ -10,12 +10,12 @@ import { renderWithTheme } from '../../util/test-utils';
 
 const phoneWidth = parseInt(viaTheme.screenSize.phone, 10);
 
-Object.defineProperty(window.document.body, 'clientWidth', {
+Object.defineProperty(window, 'innerWidth', {
   writable: true
 });
 
 function setScreenWidth(size) {
-  global.document.body.clientWidth = size;
+  window.innerWidth = size;
```

Comment:
This isn't going to cause problems since it doesn't take into account the scrollbar, will it?

Reply:
`innerWidth` does include the space that a scroll bar would take up. We can do something else if that is going to cause bugs, but then I think we run into the same problem that #467 references, which is that this does not match the media query.